docs/Xamarin.Forms.Core/Xamarin.Forms/Cell.xml

```diff
@@ -554,7 +554,7 @@ Content = new TableView
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>This method is for internal use by platform renderers.</summary>
```

Comment:
this is explicit interface implementation. I don't think it should appear in the docs